### PR TITLE
Make imported test files available to both test and test support trees

### PR DESCRIPTION
**Pull request by asakusuma**
Addresses https://github.com/ember-cli/ember-cli/issues/4982

### CODE REVIEW

**Comment by asakusuma** [comment]
Will add tests later today

**Comment by asakusuma** [comment]
An acceptance test has been added

**Comment by trentmwillis** [comment]
:+1: looks good to me! Thanks for taking the time to fix this.

**Comment by rwjblue** [comment]
@homu r+

**Comment by chadhietala** [comment]
@homu r+

**Comment by chadhietala** [comment]
It appears there was a race to approve. Hopefully things go well :unamused:

File: lib/broccoli/ember-app.js [hunk 1]
```diff
@@ -956,18 +956,26 @@ EmberApp.prototype.appAndDependencies = function() {
   });
 };
 
+EmberApp.prototype.test = function() {
```
Comment (stefanpenner) [inline lib/broccoli/ember-app.js]: missing docs

**Comment by stefanpenner** [comment]
In some of my tests this appears to be introducing some unexpected work

File: lib/broccoli/ember-app.js (hunk 1 above)
Comment (asakusuma) [inline lib/broccoli/ember-app.js]: * I can add docs
* The name is meant to be consistent with the names of the other methods feeding into toArray() (like javascript()) but I can rename to something more descriptive.

Comment (stefanpenner) [inline lib/broccoli/ember-app.js]: I suspect docs, will do the trick for the name as well.

**Comment by asakusuma** [comment]
Any update here? Are these tests public?

**Comment by stefanpenner** [comment]
@asakusuma i noticed it when looking at a LinkedIn slow-trees someone pasted me ;)

**Pull request merged**